samples/bluetooth/path_loss_monitoring/central/Kconfig.sysbuild

```
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "share/sysbuild/Kconfig"
+
+config NRF_DEFAULT_IPC_RADIO
+	default y
```

```suggestion
	default y
	
config NETCORE_IPC_RADIO_BT_HCI_IPC
	default y
```
Maybe if we add this to align with peripheral and then we can simplify the kconfigs